addons/sms/views/sms_sms_views.xml

```diff
@@ -34,11 +34,14 @@
         <field name="name">sms.sms.view.tree</field>
         <field name="model">sms.sms</field>
         <field name="arch" type="xml">
-            <tree string="SMS Templates" decoration-muted="state == 'canceled'" decoration-info="state == 'outgoing'" decoration-danger="state == 'error'">
+            <tree string="SMS Templates">
                 <field name="number"/>
                 <field name="partner_id"/>
-                <field name="state"/>
                 <field name="failure_type"/>
+                <field name="state" widget="badge" decoration-info="state == 'outgoing'" decoration-muted="state == 'canceled'" decoration-success="state == 'sent'" decoration-danger="state == 'error'"/>
```

Comment:
Could put the various decoration attributes on the next line to avoid having extra long lines.

Reply:
When I wrote this code, I checked other places where `decoration-xxx` is used and the line is really long.
I think almost all of them use one line strategy, unless the [condition](https://github.com/odoo/enterprise/blob/ad6a4dd959aa1f03324f157a2f32c96ed8c701df/mrp_plm/views/mrp_eco_views.xml#L512) is completed.
If you still think the current style is not developer-friendly, I can change it:)

Reply:
I'm not sure there is a specific guideline for this, I just saw that the line goes out of my (big) screen on github:

![image](https://user-images.githubusercontent.com/41282536/135051705-be19df0b-208a-4001-924e-8bbe55abf332.png)

That's why I suggested the multi-line.
If you don't agree it's fine, not a big deal anyway.

Reply:
Since this [line](https://github.com/odoo/enterprise/blob/83d5b60b9f5955054055e30a24e30457a8882b39/approvals/views/approval_request_views.xml#L57) is updated recently and longer than mine. I prefer to keep the current coding style:)